about.html

```diff
@@ -17,6 +17,38 @@
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/2.6.3/svg.js"></script>
 	<script src="assets/scripts/scripts.js"></script>
 	<script src="https://use.fontawesome.com/d11bbf05c5.js"></script>
+
+
+	<script>
+		function openPopup() {
+			document.getElementById("services-popup").style.display = "block";
+		}
+
+		function closePopup() {
+			document.getElementById("services-popup").style.display = "none";
+		}
+
+		function openthePopup() {
```

Comment:
why are there basically three versions of these functions?

Reply:
I tried doing it with one single function, but if they are all the same name, then whenever I open the second or third popup, it will open the first popup because the program reads it from top to bottom.